package.json

```diff
@@ -136,7 +136,7 @@
   },
   "private": true,
   "dependencies": {
-    "@bower_components/Croppie": "Foliotek/Croppie#^2.4.1",
+    "@bower_components/Croppie": "Foliotek/Croppie#^2.6.4",
```

Comment:
@maze-runnar You sure there will be no changes to `yarn.lock` for this ?

Reply:
yes , i don't made any changes beside this in my local . So i am sure .

Reply:
Revert it, not needed